ssawg_trending_scraper.py

```diff
@@ -407,6 +407,11 @@ def get_url(self):
     tb.url_html,
     tb.headers3[0],
     tb.images["one_shot_vs_angle.png"],
+    "<br><br><table><tbody><tr><td>",
```

Comment:
I think this needs `tb.headers3[1]`.  I'm guessing on the `[1]` but that is probably right. Something like `tb.paragraphs[1]` to get the `Note that in addition to filtering ...` would also be nice.

Reply:
I have no idea what any of that means.

Reply:
Oh, you want text.  Do we care?  I figured by the time we were there we'd have clicked to see the full report.  I suppose we can also just put the whole report in the trending page.

Reply:
I added the text and then through in the histograms too, because if we have a higher bar than I thought to clicking through, maybe we just want them.